Apps/WebClient/src/NewClientApp/src/components/site/FooterComponent.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import { useAuthStore } from "@/stores/auth";
+import { useConfigStore } from "@/stores/config";
+import { useUserStore } from "@/stores/user";
+
+const authStore = useAuthStore();
+const configStore = useConfigStore();
+const userStore = useUserStore();
+
+const isFooterShown = computed<boolean>(
+    () =>
+        !configStore.isOffline &&
+        (!authStore.oidcIsAuthenticated ||
+            !userStore.isValidIdentityProvider ||
+            userStore.userIsRegistered)
+);
```

Comment:
I suspect this can be removed - it simply hides the footer when the app is in offline mode (no longer used) or on the registration page.  Can follow up with Nino later.

Reply:
Will leave.